utils/docker/Dockerfile.centos.7

```diff
@@ -109,44 +140,20 @@ ENV PATH=$PATH:/home/daos/.local/bin
 
 
 # Create directory for DAOS backend storage
-RUN mkdir -p /opt/daos /mnt/daos /var/run/daos_server /var/run/daos_agent &&	\
-    chown daos_server.daos_server /opt/daos /mnt/daos /var/run/daos_server &&	\
+RUN mkdir -p /opt/daos /mnt/daos /var/run/daos_server /var/run/daos_agent &&   \
+    chown daos_server.daos_server /opt/daos /mnt/daos /var/run/daos_server &&  \
     chown daos_agent.daos_agent /var/run/daos_agent
 
 # Dependencies
 # Packages for NVML exist in CentOS, but are
 # unfortunately outdated. The DAOS build system will rebuild those packages.
+# Currently this still must use yum as the QUICKBUILD_DEPS may have a
+# trailing newline that causes DNF to fail.
```

Comment:
I think it would be better to just fix up the list so that `dnf` can use it.

Reply:
I spent a lot of time trying to fix up the list.  I asked you for advise on the matter, and since you did not have a solution, I fell back to yum.   But this means it is currently broken on CentOS 8.

Reply:
I didn't have a solution for a very generally expressed problem.  Can you show where/how it was failing, in a run perhaps to see what a specific solution could be.

Reply:
Fixed in current push